src/differentials/abstract_zero.jl

```diff
@@ -71,7 +71,7 @@ arguments.
     end
 ```
 """
-struct DoesNotExist <: AbstractZero end
+struct NoPossibleTangent <: AbstractZero end
```

Comment:
Shorter and means the same thing?
```suggestion
struct NoTangent <: AbstractZero end
```

Reply:
Arguably means a different thing, but might still be good